machida/lib/wallaroo/__init__.py

```diff
@@ -69,13 +72,22 @@ def build_application(app_name, pipeline):
     return pipeline.__to_tuple__(app_name)
 
 
+def range_windows(wrange):
```

Comment:
I wonder if it would be simpler for users to call `RangeWindowsBuilder(...)` directly?